complyscribe/transformers/cac_transformer.py

```diff
@@ -332,17 +326,22 @@ def _get_params_properties(
         )
         return [id_prop, description_prop, alternative_prop]
 
-    def _get_rule_properties(self, ruleset: str, rule_obj: RuleInfo) -> List[Property]:
+    def _get_rule_properties(
+        self, ruleset: str, rule_obj: RuleInfo, rule_index: int
+    ) -> List[Property]:
         """Get a set of rule properties for a rule object."""
         rule_properties: List[Property] = []
         # Add rule properties for the ruleset
         rule_properties.append(add_prop(RULE_ID, rule_obj.id, ruleset))
         rule_properties.append(
             add_prop(RULE_DESCRIPTION, rule_obj.description, ruleset)
         )
-        for index, param in enumerate(rule_obj._parameters):
-            suffix = "" if len(rule_obj._parameters) == 1 else f"_{index}"
-            rule_properties.extend(self._get_params_properties(ruleset, param, suffix))
+        if rule_index == 0:
```

Comment:
I would only suggest to include a comment here providing some context on why we are including parameters only for the first rule. We could link the Jira ticket so in the future we can easily get the context. WDYT?

Reply:
@marcusburghardt Thank you. Updated.